src/sched/job_runner.h

```diff
@@ -57,6 +65,20 @@ class JobRunner : public std::enable_shared_from_this<JobRunner> {
 
     bool AddJob(std::function<void(JobAliveTokenPtr&&)>&& job, JobRunnerStrandPtr strand);
 
+    ///
```

Comment:
remove this line?

Reply:
Merge discussion into https://github.com/cyfitech/cris-core/pull/121#discussion_r1059309471